code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -49,6 +49,14 @@
 	icon_state = "tile-wood"
 	turf_type = /turf/simulated/floor/wood
 
+/obj/item/stack/tile/wood/cyborg

```

Comment:
@tigercat2000 I'll get rid of them, but why does any cyborg material exist then?